common/workunit/workflow.hpp

```diff
@@ -26,9 +26,9 @@
 #include <thread>
 #include <atomic>
 
-#ifdef _DEBUG
-    #define TRACE_WORKFLOW
-#endif
+//#ifdef _DEBUG
```

Comment:
the #if and the #endif don't need to be commented out.  Then you just need to uncomment a single line to enable it